requirements/bench.txt

```diff
@@ -14,3 +14,4 @@ torchvision
 pyomo>=5.7
 IOHexperimenter>=0.2.8
 mixsimulator>=0.2.9.5
+cdt
```

Comment:
can you please add the current version as the minimal version to use? this is sometimes useful to identify easily which was a working version when code breaks